README.md

```diff
@@ -40,6 +40,16 @@ If you use Picasso, let it use your `OkHttpClient`, and you'll get RUM and APM i
         Picasso.setSingletonInstance(picasso)
 ```
 
+### Integrating with Retrofit
```

Comment:
If you're OK with that, I'll add this to my documentation branch to avoid you having to rebase and solve conflicts here.

Reply:
Perfect...will drop this commit then